src/ow1/heroes/bastion.opy

```diff
@@ -281,6 +281,14 @@ rule "[bastion.opy]: Stop self-repair on ability 2 release":
     eventPlayer.self_repair = false
 
 
+rule "[bastion.opy]: Stop self-repair and hide its GUI on death":
+    @Event playerDied
+    @Hero bastion
+
+    eventPlayer.self_repair = false
+    eventPlayer.heal_gui_visibility = null
```

Comment:
Delete this line because not healing yourself doesn't necessarily imply not seeing heal gui. Hiding the gui might result in the gui now showing up the next time bastion heals himself.

Also I'm fairly certain that you still see the heal gui recharging if you die while trying to heal yourself.
```suggestion
```

Reply:
> not healing yourself doesn't necessarily imply not seeing heal gui

Is seeing the heal GUI after death intended behaviour?

> Hiding the gui might result in the gui not showing up the next time bastion heals himself.

I cannot reproduce this on my end. I tried: healing while in the process of dying, healing after dying, healing shortly and dying once the heal GUI is gone, healing and then stop healing before dying, and all of the above with a Mercy resurrect after. Can't think of any edge case where this can occur.

> Also I'm fairly certain that you still see the heal gui recharging if you die while trying to heal yourself.

This is what the line you asked me to remove does. It ensures it cannot be seen after death.

Reply:
Ok I’ll approve and merge to staging for now, if you say it works.